src/components/VocabularyListItem.tsx

```diff
@@ -79,17 +78,19 @@ const VocabularyListItem = ({ document }: VocabularyListItemPropType) => {
     <View style={styles.wrapper}>
       <View style={styles.container}>
         <View style={styles.item}>
-          <Image
-            source={{
-              uri: document.document_image[0].image
-            }}
-            width={24}
-            height={24}
-            style={styles.image}
-          />
+          {document.document_image.length > 0 && (
+            <Image
+              source={{
+                uri: document?.document_image[0].image
```

Comment:
```suggestion
                uri: document.document_image[0].image
```
as the check is not used in line 81 as well.

Reply:
done